proto/elys/parameter/tx.proto

```diff
@@ -79,3 +82,25 @@ message MsgUpdateRewardsDataLifetime {
 }
 
 message MsgUpdateRewardsDataLifetimeResponse {}
+
+message MsgUpdateTakerFees {
+  option (cosmos.msg.v1.signer) = "creator";
+  option (amino.name) = "parameter/MsgUpdateTakerFees";
+  string creator = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+  string taker_fees = 2 [
+    (cosmos_proto.scalar) = "cosmos.Dec",
+    (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
+    (gogoproto.nullable) = false
+  ];
+}
+
+message MsgUpdateTakerFeesResponse {}
+
+message MsgUpdateTakerFeeCollectionAddress {
```

Comment:
lets not have this, it should always be module address, fixed (unless there is token economics change)